metapool/prep.py

```diff
@@ -75,6 +75,8 @@
                'Machine type': 'HiSeq/RR', 'run_center': 'IGM'},
     'LH00444': {'machine prefix': 'LH', 'Vocab': 'Illumina NovaSeq X',
                 'Machine type': 'NovaSeq', 'run_center': 'IGM'},
+    'SH00252': {'machine prefix': 'SH', 'Vocab': 'Illumina MiSeq i100',
+                'Machine type': 'MiSeq i100', 'run_center': 'IGM'},
```

Comment:
Bloody heck, I didn't even know there was a list of sequencers *here*.  Do you know if this list is referenced in qp-klp or any other code outside of kl-metapool?

Per issue #234 , we also have sequencer lists in (at least?) two other places.  Am I correct in understanding that we don't yet know what the barcode rc/etc requirements are for the MiSeq i100 so we are not sure how to add it to those places? Once we know those answers, I think now is the time for me to actually fix #234 because this is out of hand -_-

Reply:
I had no idea about issue #234 and after looking in the code the only place that it is used is metapool.prep.get_model_and_center, which is not used anywhere. How do you suggest we proceed?